moveit2.repos

```diff
@@ -1,28 +1,8 @@
 repositories:
-  moveit_msgs:
-    type: git
-    url: https://github.com/ros-planning/moveit_msgs
-    version: ros2
-  moveit_resources:
-    type: git
-    url: https://github.com/ros-planning/moveit_resources
-    version: ros2
   geometric_shapes:
     type: git
     url: https://github.com/ros-planning/geometric_shapes
     version: ros2
-  srdfdom:
-    type: git
-    url: https://github.com/ros-planning/srdfdom
-    version: ros2
-  ros2_control:
-    type: git
-    url: https://github.com/ros-controls/ros2_control
-    version: master
-  ros2_controllers:
-    type: git
-    url: https://github.com/ros-controls/ros2_controllers
-    version: master
```

Comment:
We should not need the .repos files but as per idea of @tylerjw and discussion [here](https://github.com/ros-planning/moveit2/pull/565) we left them unused . If in a case some PR depends on one of these repositories. We could re-enable the .repos in CI.

Reply:
My idea in this PR is to list basic upstream dependencies in `moveit2.repos` and augment these with additional packages required for older distro, i.e. via `moveit2_galactic.repos`.